website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,280 @@
-            [--source {YahooFinance,AlphaVantage,Polygon,EODHD,Intrinio}]
-
-Load stock ticker to perform analysis on. When the data source is yf, an Indian ticker can be loaded by using '.NS' at the end, e.g. 'SBIN.NS'. See available market in https://help.yahoo.com/kb/exchanges-data-
-providers-yahoo-finance-sln2310.html.
-
-optional arguments:
-  -t TICKER, --ticker TICKER
-                        Stock ticker (default: None)
-  -s START, --start START
-                        The starting date (format YYYY-MM-DD) of the stock (default: 2020-02-11)
-  -e END, --end END     The ending date (format YYYY-MM-DD) of the stock (default: 2023-02-15)
-  -i {1,5,15,30,60}, --interval {1,5,15,30,60}
-                        Intraday stock minutes (default: 1440)
-  -p, --prepost         Pre/After market hours. Only reflected in 'YahooFinance' intraday data. (default: False)
-  -f FILEPATH, --file FILEPATH
-                        Path to load custom file. (default: None)
-  -m, --monthly         Load monthly data (default: False)
-  -w, --weekly          Load weekly data (default: False)
-  --exchange            Show exchange information. (default: False)
-  --performance         Show performance information. (default: False)
-  -h, --help            show this help message (default: False)
-  --export EXPORT       Export raw data into csv, json, xlsx (default: )
+options:
+  -t TERM [TERM ...], --term TERM [TERM ...]
+                        search for a term on the news
+  -s SOURCES, --sources SOURCES
+                        sources from where to get news from (separated by comma)
+  -h, --help            show this help message
+  --export EXPORT       Export raw data into csv, json, xlsx
   --sheet-name SHEET_NAME [SHEET_NAME ...]
-                        Name of excel sheet to save data to. Only valid for .xlsx files. (default: None)
-  --source {YahooFinance,AlphaVantage,Polygon,EODHD,Intrinio}
-                        Data source to select from (default: YahooFinance)
-
-For more information and examples, use 'about load' to access the related guide.
+                        Name of excel sheet to save data to. Only valid for .xlsx files.
+  -l LIMIT, --limit LIMIT
+                        Number of entries to show in data.
 
+For more information and examples, use 'about news' to access the related guide.
 ```
 
-This shows you all **arguments** the command has. These are additional options you can provide to the command. Each default value is also displayed which is used when you do not select this option. For example, if I would use the <a href="https://www.investopedia.com/ask/answers/12/what-is-a-stock-ticker.asp" target="_blank" rel="noreferrer noopener">stock ticker</a> of Amazon (AMZN, which can also be found with `search amazon`), I can use `load AMZN` which will return the following:
+Try:
```

Comment:
Maybe expand on this:  "To search for news contained the term Federal Reserve, you can try the command:"